libs/shared/type-site/src/lib/i-component.ts

```diff
@@ -64,6 +64,7 @@ export type IComponentState = string | number | boolean | Array<unknown> | objec
 export type IEventParams = Record<string, IEventParamEntry>
 
 export type IEventParamEntry = {
+  type: ComponentArgType
```

Comment:
Event param type was added so that we can use it to display checkbox for boolean params.

Reply:
This makes sense, although technically we would need to migrate existing sites, since it's a new required field. One way around it is to make `type` optional, though that could be annoying.

In this case I don't think anything will break when `type` is empty, but in the future we can add a process for breaking changes to the data format (e.g. add code for migrating during/after deserialization, and increment the site format version)